packages/forms/src/FormikCheckboxField.tsx

```diff
@@ -41,7 +46,11 @@ export function FormikCheckboxField({
       }}
       onChange={(event, checked) => {
         onChange?.(event, checked);
-        field.onChange(event);
+        if (parse) {
+          setValue(parse(event as ChangeEvent<HTMLInputElement>));
```

Comment:
well, that's not right too. we should respect upstream types.

Reply:
Material marked the event as `ChangeEvent<{}>`